integration/qbft/tests/all_test.go

```diff
@@ -1,9 +1,9 @@
 package tests
 
 import (
+	"github.com/bloxapp/ssv-spec/types"
```

Comment:
please revert this change

Reply:
![image](https://user-images.githubusercontent.com/121814400/211524166-fb99cf6b-3379-434a-a87f-a3953cf0f321.png)

Reply:
@egorka-sereda This line is needed, but you just moved it from the block with 3rd party imports to the block with standard library imports, which made imports unsorted. Actually, not a big deal, but I'd prefer not to do this change